src/js/gi/components/If.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class If extends React.Component {
```

Comment:
Rather than create an `If` component, I think it's more idiomatic to either (1) conditionally assign JSX to variables or (2) just inline the conditions. https://facebook.github.io/react/docs/conditional-rendering.html

```
// Conditionally assigning JSX.

let conditionalSection;

if (condition) {
  conditionalSection = <div>This section appears because of condition.</div>;
}

return (
  <div>
    <span>This always appears.</span>
    {conditionalSection}
  </div>
);
```

```
// Inlining conditional sections.

function renderConditionalSection() {
  return <div>This section appears because of condition.</div>;
}

function render() {
  return (
    <div>
      <span>This always appears.</span>
      {condition && renderConditionalSection()}
    </div>
  );
}
```

From [this SO answer](http://stackoverflow.com/a/25225016): "As for the If component you asked about, one problem it has is that in its current form it will evaluate its children even if the condition is false. This can lead to errors when the body of the If only makes sense if the condition is true."